frontend/src/pages/AccountSend.vue

```diff
@@ -1206,18 +1209,11 @@ function useSendForm() {
     );
   }
 
-  function checkConfusables() {
-    const recipientIdString = recipientId.value || '';
-    try {
-      if (recipientIdString && recipientIdString.endsWith('eth')) {
-        assertValidEnsName(recipientIdString);
-      }
-      recipientIdWarning.value = undefined;
-    } catch (e) {
-      if (e instanceof Error) {
-        recipientIdWarning.value = e.message;
-      }
+  function checkConfusables(recipientIdString: string | undefined) {
+    if (recipientIdString && recipientIdString.endsWith('eth')) {
+      assertValidEnsName(recipientIdString);
     }
+    recipientIdWarning.value = undefined;
```

Comment:
Is this needed? Will this supress all warnings?

Reply:
Gone: [34740c2](https://github.com/ScopeLift/umbra-protocol/pull/640/commits/34740c2777b044c67badac8536a056d80aadd2ca)